UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Logging/EventIds.cs

```diff
@@ -653,7 +653,51 @@ public enum EventIds
         /// <summary>
         /// 809129 - Products Fetched from ESS.
         /// </summary>
-        ProductsFetchedFromESS = 809129
+        ProductsFetchedFromESS = 809129,
+        /// <summary>
+        /// 809130 - Request for searching readme.txt from file share service is started.
+        /// </summary>
+        QueryFileShareServiceReadMeFileRequestStart = 809130,
+
+        /// <summary>
+        /// 809131 - Request for searching readme.txt from file share service is completed.
+        /// </summary>
+        QueryFileShareServiceReadMeFileRequestCompleted = 809131,
+
+        /// <summary>
+        /// 809132 - Request for searching readme.txt from file share service is failed due to non ok response.
+        /// </summary>
+        QueryFileShareServiceReadMeFileNonOkResponse = 809132,
+
+        /// <summary>
+        /// 809133 - Readme.txt file is not found while searching in file share service.
+        /// </summary>
+        ReadMeTextFileNotFound = 809133,
+
+        /// <summary>
+        /// 809134 - Request for downloading readme.txt from file share service is started.
+        /// </summary>
+        DownloadReadMeFileRequestStart = 809134,
+
+        /// <summary>
+        /// 809135 - Request for downloading readme.txt from file share service is completed.
+        /// </summary>
+        DownloadReadMeFileRequestCompleted = 809135,
+
+        /// <summary>
+        /// 809136 - Request for downloading readme.txt from file share service is failed due to non ok response.
+        /// </summary>
+        DownloadReadMeFileNonOkResponse = 809136,
+
+        /// <summary>
+        /// 809137 - Completed download of 307 response README files from the file share service.
+        /// </summary>
+        DownloadReadmeFile307RedirectResponse = 809137,
+
+        /// <summary>
+        /// 809138 - Multiple files found for README files from the file share service.
+        /// </summary>
+        QueryFileShareServiceMultipleFilesFound = 809138
```

Comment:
```suggestion
        /// <summary>
        /// 809137 - Completed download of 307 response readme.txt file from the file share service.
        /// </summary>
        DownloadReadMeFile307RedirectResponse = 809137,

        /// <summary>
        /// 809138 - Multiple files found for readme.txt from the file share service.
        /// </summary>
        QueryFileShareServiceMultipleReadMeFilesFound = 809138
```

Reply:
Resolved .
For this eventId "QueryFileShareServiceMultipleFilesFound" there will be any multiple files will get from fss so accordingly given name